Sluggo/View Controllers/Extensions/ViewControllerExtensions.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ViewControllerExtensions.swift
+//  Sluggo
+//
+//  Created by Isaac Trimble-Pederson on 5/10/21.
+//
+
+import UIKit
+
+extension UIViewController {
+    func presentError(error: Error, completion: ((UIAlertAction) -> Void)?) {
+        UIAlertController.createAndPresentError(vc: self, error: error, completion: completion)
+    }
+    
+    func presentError(error: Error) {
+        UIAlertController.createAndPresentError(vc: self, error: error)
+    }
+    
+    func processResult<T>(result: Result<T, Error>,  onSuccess: @escaping ((T) -> Void),  onError: ((Error) -> Void)?,  after: (() -> Void)?) {
```

Comment:
Not really sure I understand what the difference is between onSuccess and after is. Is this equivalent to a "finally" block in Javascript promises land?